<ui version="4.0" >
 <class>Settings</class>
 <widget class="QDialog" name="Settings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QEN - Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="Settings.qrc" >:/apps/Images/apps/infolog.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>260</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QPushButton" name="pbAccept" >
      <property name="text" >
       <string>Accept</string>
      </property>
      <property name="icon" >
       <iconset resource="Settings.qrc" >:/actions/Images/actions/button_ok.png</iconset>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QPushButton" name="pbCancel" >
      <property name="text" >
       <string>Cancel</string>
      </property>
      <property name="icon" >
       <iconset resource="Settings.qrc" >:/actions/Images/actions/button_cancel.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="cbServerURL" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cbLogin" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cbPassword" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>140</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="cbRefresh" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>170</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbAppearance" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Appearance</string>
   </property>
   <widget class="QCheckBox" name="cbARC" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>167</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Display alternating row colours in the network/peer listbox&lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>Alternating Row Colours</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="cbDBPath" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>200</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cbDBPathLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Database path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cbRefreshLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Refresh Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="cbPasswordLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>150</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cbServerURLLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Server URL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cbLoginLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Login:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pbAccept</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="Settings.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbAccept</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
